<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1201</width>
    <height>690</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1201</width>
    <height>690</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1201</width>
    <height>690</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="stackTaskList">
    <property name="geometry">
     <rect>
      <x>209</x>
      <y>110</y>
      <width>731</width>
      <height>501</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QTreeWidget" name="taskList">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>701</width>
        <height>491</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string/>
       </property>
      </column>
      <column>
       <property name="text">
        <string>到期时间</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>提醒时间</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>创建时间</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>描述</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>备注</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string/>
       </property>
      </column>
      <item>
       <property name="text">
        <string>未安排</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
      </item>
      <item>
       <property name="text">
        <string>未来七天</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
      </item>
      <item>
       <property name="text">
        <string>以后</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
      </item>
      <item>
       <property name="text">
        <string>已逾期</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
      </item>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QTreeWidget" name="taskList_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>701</width>
        <height>491</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string/>
       </property>
      </column>
      <column>
       <property name="text">
        <string>到期时间</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>提醒时间</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>创建时间</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>描述</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>备注</string>
       </property>
      </column>
      <item>
       <property name="text">
        <string>待完成</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
      </item>
      <item>
       <property name="text">
        <string>显示已完成</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
      </item>
     </widget>
    </widget>
   </widget>
   <widget class="QStackedWidget" name="taskSetting">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>40</y>
      <width>251</width>
      <height>571</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page_3"/>
    <widget class="QWidget" name="page_4">
     <widget class="Line" name="line_5">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>100</y>
        <width>251</width>
        <height>20</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QComboBox" name="prioritySet">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>无优先级</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>优先级低</string>
       </property>
       <property name="icon">
        <iconset>
         <normalon>:/icons/icon/lowPriority.png</normalon>
        </iconset>
       </property>
      </item>
      <item>
       <property name="text">
        <string>优先级中</string>
       </property>
       <property name="icon">
        <iconset>
         <normalon>:/icons/icon/midPriority.png</normalon>
        </iconset>
       </property>
      </item>
      <item>
       <property name="text">
        <string>优先级高</string>
       </property>
       <property name="icon">
        <iconset>
         <normalon>:/icons/icon/highPriority.png</normalon>
        </iconset>
       </property>
      </item>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>124</y>
        <width>131</width>
        <height>341</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="setDuetime">
         <property name="text">
          <string>设置到期时间</string>
         </property>
         <property name="icon">
          <iconset>
           <normalon>:/icons/icon/setDueTime.png</normalon>
          </iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="setRemindtime">
         <property name="text">
          <string>设置提醒</string>
         </property>
         <property name="icon">
          <iconset>
           <normalon>:/icons/icon/setRemindTime.png</normalon>
          </iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="setDescription">
         <property name="text">
          <string>添加描述</string>
         </property>
         <property name="icon">
          <iconset>
           <normalon>:/icons/icon/setDescription.png</normalon>
          </iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="setNote">
         <property name="text">
          <string>添加备注</string>
         </property>
         <property name="icon">
          <iconset>
           <normalon>:/icons/icon/setNote.png</normalon>
          </iconset>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QStackedWidget" name="stackEditTitle">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>231</width>
        <height>51</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page_7">
       <widget class="QLineEdit" name="modifyTaskTitle">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>10</y>
          <width>181</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_8">
       <widget class="QPushButton" name="modifyTaskTitleButton">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>10</y>
          <width>33</width>
          <height>29</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normalon>:/icons/icon/edit.png</normalon>
         </iconset>
        </property>
       </widget>
       <widget class="QCheckBox" name="taskTitle">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>171</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>20</y>
      <width>1231</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>610</y>
      <width>1201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>30</y>
      <width>20</width>
      <height>591</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>30</y>
      <width>20</width>
      <height>591</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>171</width>
      <height>571</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="MyFrontPageButton">
         <property name="text">
          <string>我的首页</string>
         </property>
         <property name="icon">
          <iconset>
           <normalon>:/icons/icon/myFrontPage.png</normalon>
          </iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="MyTaskPageButton">
         <property name="text">
          <string>我的待办</string>
         </property>
         <property name="icon">
          <iconset>
           <normalon>:/icons/icon/myTask.png</normalon>
          </iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QListWidget" name="groupList">
       <item>
        <property name="text">
         <string>添加分组</string>
        </property>
        <property name="icon">
         <iconset>
          <normalon>:/icons/icon/addGroup.png</normalon>
         </iconset>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QStackedWidget" name="AddTask">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>80</y>
      <width>721</width>
      <height>31</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page_5">
     <widget class="QPushButton" name="clickToAddTask">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>701</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>+点击新建待办</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_6">
     <widget class="QLineEdit" name="InputToAddTask">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>711</width>
        <height>31</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>输入内容后，回车创建成功</string>
      </property>
     </widget>
     <widget class="QPushButton" name="chooseDate">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>0</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normalon>:/icons/icon/calendar.png</normalon>
       </iconset>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>50</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>我的首页</string>
    </property>
   </widget>
   <widget class="QPushButton" name="groupRefreshButton">
    <property name="geometry">
     <rect>
      <x>825</x>
      <y>51</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>刷新</string>
    </property>
    <property name="icon">
     <iconset>
      <normalon>:/icons/icon/refresh.png</normalon>
     </iconset>
    </property>
   </widget>
   <widget class="QComboBox" name="sortBox">
    <property name="geometry">
     <rect>
      <x>646</x>
      <y>50</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>排序</string>
     </property>
     <property name="icon">
      <iconset>
       <normalon>:/icons/icon/setDueTime.png</normalon>
      </iconset>
     </property>
    </item>
    <item>
     <property name="text">
      <string>到期时间排序</string>
     </property>
     <property name="icon">
      <iconset>
       <normalon>:/icons/icon/sort.png</normalon>
      </iconset>
     </property>
    </item>
    <item>
     <property name="text">
      <string>创建时间排序</string>
     </property>
     <property name="icon">
      <iconset>
       <normalon>:/icons/icon/sortByCreateTime.png</normalon>
      </iconset>
     </property>
    </item>
    <item>
     <property name="text">
      <string>优先级排序</string>
     </property>
     <property name="icon">
      <iconset>
       <normalon>:/icons/icon/sortByPriority.png</normalon>
      </iconset>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="refreshButton">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>50</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>刷新</string>
    </property>
    <property name="icon">
     <iconset>
      <normalon>:/icons/icon/refresh.png</normalon>
     </iconset>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1201</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
